app/models/product.rb

```diff
@@ -5,26 +5,28 @@
 #  id          :bigint           not null, primary key
 #  deleted_at  :datetime
 #  description :string
+#  enabled     :boolean          default(TRUE), not null
 #  name        :string           not null
 #  created_at  :datetime
 #  updated_at  :datetime
 #
 # Indexes
 #
 #  index_products_on_deleted_at  (deleted_at)
+#  index_products_on_enabled     (enabled)
 #  index_products_on_name        (name)
 #
 
 class Product < ApplicationRecord
   acts_as_paranoid
-
+  scope :enabled, lambda {
```

Comment:
This should go at the end. Take a look at [this](https://github.com/rootstrap/tech-guides/blob/master/ruby/rails.md#macro-style-methods)